src/nbl/video/CVulkanLogicalDevice.h

```diff
@@ -1276,17 +1276,21 @@ class CVulkanLogicalDevice final : public ILogicalDevice
         core::smart_refctd_ptr<IGPUDescriptorSetLayout>&& layout3 = nullptr) override
     {
         constexpr uint32_t MAX_PC_RANGE_COUNT = 100u;
-        constexpr uint32_t MAX_DESCRIPTOR_SET_LAYOUT_COUNT = 4u;
+
+        core::smart_refctd_ptr<IGPUDescriptorSetLayout> dummyDSLayout = nullptr;
+        if (!layout0 || !layout1 || !layout2 || !layout3)
+            dummyDSLayout = createGPUDescriptorSetLayout_impl(nullptr, nullptr);
```

Comment:
dont create it as a `IGPUDescriptorSetLayout` just have it a plain `vkDescrptorSetLayout`

Reply:
invalid and bad advice... resolving